.travis.yml

```diff
@@ -54,15 +54,14 @@ jobs:
       language: shell
 
     - name: "Deploy PyPI package"
-      install: echo "Skipping install stage"
+      install:
```

Comment:
I don't think installing is necessary, but at least I'm sure this creates a version file. Not sure what the `deploy` stage with `provider: pypi` does exactly.

Reply:
Where does it create the version file in the install location?

Reply:
https://github.com/ComPWA/expertsystem/blob/4fc2df6c21dd0c9fe611144b1de8d01b77a238c0/pyproject.toml#L9-L10

Reply:
can't you test that the version will be correct locally. In addition you can also use https://test.pypi.org/ to upload

Reply:
Thing is that I'm not sure what this `deploy` job does exactly on Travis.

Reply:
well build the sdist and upload it to pypi

Reply:
> This feature requires Setuptools 42 or later, released in Nov, 2019. **If your project needs to support build from sdist on older versions of Setuptools, you will need to also implement the `setup.py usage` for those legacy environments.**

Not sure what Travis does it you remove `sdist`

Reply:
> well build the sdist and upload it to pypi

but yeah `sdist` does not work, also locally, it does not produce the version file

Reply:
So does the current code work? I want to avoid doing trail and error here

Reply:
No, also not `bdist_wheel`

Reply:
Should work now :crossed_fingers: